### PR TITLE
enable npm install on post-create and add DEV-GUIDE.md v1

### PR DESCRIPTION
**Pull request by hazamashoken**
- add dev-guide documentation for people starting
- enable "npm install" automatically on post-create

### CODE REVIEW

File: .devcontainer/devcontainer.json [hunk 1]
```diff
@@ -22,14 +20,11 @@
...
     }
   },
-
   // Use 'forwardPorts' to make a list of ports inside the container available locally.
   // "forwardPorts": [3000, 27017],
-
   // Use 'postCreateCommand' to run commands after the container is created.
-  // "postCreateCommand": "yarn install",
-
+  "postCreateCommand": "npm install",
```
Comment (Lodimup) [inline .devcontainer/devcontainer.json:26]: cd app && npm install

File: .devcontainer/devcontainer.json [hunk 2]
```diff
@@ -22,14 +20,11 @@
...
   // Uncomment to connect as root instead. More info: https://aka.ms/dev-containers-non-root.
   // "remoteUser": "root"
   "remoteUser": "root"
```
Comment (Lodimup) [inline .devcontainer/devcontainer.json:29]: let's not do this on pub

**Comment by hazamashoken** [comment]
fixed

**Comment by hazamashoken** [comment]
Will close this since we have change to drizzle

**Pull request closed without merging**